src/main/resources/templates/haskell/test/readme.md

```diff
@@ -0,0 +1,12 @@
+## Test Repository instructions
+
+#### Test tool & command
+Tests will be run using pytest.
```

Comment:
Is this really pytest? I'm not familiar with it, but I could not find anything related to pytest.

Reply:
No, this is a copy paste error. They use a tool which is called stack